snuba/migrations/snuba_migrations/outcomes/0003_outcomes_add_category_and_quantity.py

```diff
@@ -0,0 +1,168 @@
+from typing import Sequence
```

Comment:
I wasn't sure how much to abstract and reduce duplication in this migration -- erred on the side of less helper variables, etc. Also, for the backwards migration, I wasn't sure if i should make the relevant parts on the old migration exportable and use them here, or if redefining here is fine (what i did).